api/src/main/resources/client_api.yaml

```diff
@@ -179,7 +186,33 @@ definitions:
       orderBy:
         description: >
           An array of columns to sort the resulting data by, taken from the table specified above,
-          each one optionally followed by " DESC" for descending sort order. (Default varies by table.)
+          each one optionally enclosed in "DESCENDING()" for descending sort order.
+          (Default sort order is person_id followed by the ID of the specified table.)
+        type: array
+        items:
+          type: string
+
+  AnnotationQuery:
+    type: object
+    description: >
+      A specification for retrieving annotation values or review statuses entered for participants.
+    properties:
+      annotations:
+        description: >
+          An array of names of annotations to retrieve about participants, "review_status" for
+          the cohort review status of the participant, or "person_id" for the
+          ID of the participant. Defaults to "person_id", "review_status", and the names of all
+          defined annotations in the cohort review. This is only valid in combination with a named
```

Comment:
nit: "only valid" implies to me that the request should fail if the preconditions are not met. Seems like we might want to return an error in this case, rather than 0 results.

Reply:
So I think it's arguably correct behavior to return an empty set of results when no cohort review has been defined... because you might wind up using a notebook that works with reviews in a workspace where you haven't created a cohort review.

Whereas I think it's incorrect behavior to use annotationQuery without using cohortName, since it only makes sense to ask for annotations / review status on named cohorts that could have a review associated with them.

I updated the documentation accordingly.